machAeroTutorials/airfoilopt_multipoint.rst

```diff
@@ -66,12 +89,14 @@ The lines that require a call to the an AeroProblem is now put into a for-loop t
     :start-after: # rst funcs (beg)
     :end-before: # rst funcs (end)
 
-In the ``objCon`` function, the :math:`c_L` constraint is also placed into the for-loop.
+In the ``objCon`` function, the :math:`c_l` constraint is also placed into the for-loop.
```

Comment:
Is it worth to clarify that while we have a single obj, there are two separate Cl constraints because of the different `ap.name` suffix? (EDIT: it's clarified below, but maybe we can still refer to the naming approach?)